torch/csrc/autograd/python_variable.cpp

```diff
@@ -1692,7 +1725,11 @@ static void clear_slots(PyTypeObject* type, PyObject* self) {
   PyMemberDef* mp;
 
   n = Py_SIZE(type);
+#if PY_VERSION_HEX >= 0x030B0000
+  mp = type->tp_members;
```

Comment:
This can be used for all python versions no?

Reply:
Right, so condition removed in this case.